exchanges/execute/README.md

```diff
@@ -0,0 +1,102 @@
+<h2 align="center">@urql/exchange-execute</h2>
+<p align="center">
+<strong>An exchange for executing queries against a local schema in <code>urql</code></strong>
+<br /><br />
+<a href="https://npmjs.com/package/@urql/exchange-graphcache">
+  <img alt="NPM Version" src="https://img.shields.io/npm/v/@urql/exchange-graphcache.svg" />
+</a>
+<a href="https://bundlephobia.com/result?p=@urql/exchange-graphcache">
+  <img alt="Minified gzip size" src="https://img.shields.io/bundlephobia/minzip/@urql/exchange-graphcache.svg?label=gzip%20size" />
+</a>
+<a href="https://github.com/FormidableLabs/urql-exchange-graphcache#maintenance-status">
+  <img alt="Maintenance Status" src="https://img.shields.io/badge/maintenance-active-green.svg" />
+</a>
+<a href="https://spectrum.chat/urql">
+  <img alt="Spectrum badge" src="https://withspectrum.github.io/badge/badge.svg" />
+</a>
```

Comment:
These seem out-of-date and copied from `@urql/exchange-graphcache`. We can also just remove them, since most exchanges readmes don't need this information as long as they'll be documented in the future in the docs, e.g. https://github.com/FormidableLabs/urql/commit/b8a7548e9b66f5028d30137e872088b8911bd6aa#diff-9190c36c53f94a10dac5a70ca4510a03R1

Reply:
Removed.